src/app/services/views/ServiceListView.vue

```diff
@@ -34,6 +46,7 @@
 </template>
 
 <script lang="ts" setup>
+import { KButton } from '@kong/kongponents'
```

Comment:
All this change will be undone before merge.

Reply:
I've decided to leave this here as I am currently working on these views and I may as well leave it in here instead of remove here and re-add back in again.